bokehjs/test/models/plots/plot.coffee

```diff
@@ -190,6 +190,35 @@ describe "Plot.View render", ->
     @test_plot_view.render()
     expect(spy.calledOnce).to.be.true
 
+  it "canvas.ctx attribute should exist to allow tools to grab the DOM canvas", ->
+    expect(@test_plot_view.canvas_view.ctx).to.exist
+
+
+describe "Plot.View get_canvas_element", ->
+
+  afterEach ->
+    utils.unstub_canvas()
+    utils.unstub_solver()
+
+  beforeEach ->
+    utils.stub_canvas()
+    utils.stub_solver()
+
+    @test_doc = new Document()
+    @test_plot = new Plot({
+      x_range: new Range1d({start: 0, end: 1})
+      y_range: new Range1d({start: 0, end: 1})
+    })
+    @test_plot.document = @test_doc
+    @test_plot._doc_attached()
+    @test_plot_view = new @test_plot.default_view({ 'model': @test_plot })
+
+  it "should exist because get_canvas_element depends on it", ->
+    expect(@test_plot_view.canvas_view.ctx).to.exist
+
+  it "should exist to grab the canvas DOM element using canvas_view.ctx", ->
+    expect(@test_plot_view.get_canvas_element).to.exist
```

Comment:
You haven't actually tested the method you created.

You should setup a canvas, attach it to the plot view and then assert that the return value of `get_canvas_element()` is the canvas_view.ctx.canvas.